examples/BlueNRG_MS_SensorDemo/BlueNRG_MS_SensorDemo

```diff
@@ -0,0 +1,146 @@
+  SCLK: IDB0XA1_PIN_SPI_SCK
+  */
+SPIClass BTLE_SPI(IDB0XA1_PIN_SPI_MOSI, IDB0XA1_PIN_SPI_MISO, IDB0XA1_PIN_SPI_SCK);
+
+#define IDB0XA1_PIN_SPI_nCS    (A1)
+#define IDB0XA1_PIN_SPI_RESET  (7)
+#define IDB0XA1_PIN_SPI_IRQ    (A0)
+
+// Configure BTLE pins
+SPBTLERFClass BTLE(&BTLE_SPI, IDB0XA1_PIN_SPI_nCS, IDB0XA1_PIN_SPI_IRQ, IDB0XA1_PIN_SPI_RESET);
+
+const char *name = "BlueNRG";
+uint8_t SERVER_BDADDR[] = {0x12, 0x34, 0x00, 0xE1, 0x80, 0x03};
+
+AxesRaw_t axes_data;
+uint32_t previousSecond = 0;
+
+void setup() {
+  int ret;
+
+  Serial.begin(9600);
+
+  if(BTLE.begin() == SPBTLERF_ERROR)
+  {
+    Serial.println("Bluetooth module configuration error!");
+    while(1);
+  }
+
+  if(SensorService.begin(name, SERVER_BDADDR))
+  {
+    Serial.println("Sensor service configuration error!");
+    while(1);
+  }
+
+  /* Configure the User Button in GPIO Mode */
+  pinMode(USER_BTN, INPUT);
+
+  ret = SensorService.Add_Acc_Service();
+
+  if(ret == BLE_STATUS_SUCCESS)
+    Serial.println("Acc service added successfully.");
+  else
+    Serial.println("Error while adding Acc service.");
+
+  ret = SensorService.Add_Environmental_Sensor_Service();
+
+  if(ret == BLE_STATUS_SUCCESS)
+    Serial.println("Environmental Sensor service added successfully.");
+  else
+    Serial.println("Error while adding Environmental Sensor service.");
+
+  randomSeed(analogRead(A0));
+
+  /* Instantiate Timer Service with two characteristics:
+   * - seconds characteristic (Readable only)
+   * - minutes characteristics (Readable and Notifiable )
+   */
+  ret = SensorService.Add_Time_Service();
+
+  if(ret == BLE_STATUS_SUCCESS)
+    Serial.println("Time service added successfully.");
+  else
+    Serial.println("Error while adding Time service.");
+}
+
+void loop() {
+  BTLE.update();
+
+  if(SensorService.isConnected() == TRUE)
+  {
+    //Update accelerometer values
+    User_Process(&axes_data);
+
+    //Update time
+    SensorService.Update_Time_Characteristics();
+
+    if((millis() - previousSecond) >= 1000)
+    {
+      //Update environnemental data
+      //Data are set with random values but can be replace with data from sensors.
+      previousSecond = millis();
+      SensorService.Temp_Update(random(-100,400));
+      SensorService.Press_Update(random(95000,105000));
+      SensorService.Humidity_Update(random(0,100));
+    }
+  }
+  else
+  {
+    //Keep the Bluetooth module in discoverable mode
+    SensorService.setConnectable();
+  }
+}
+
+/**
+ * @brief  Process user input (i.e. pressing the USER button on Nucleo board)
+ *         and send the updated acceleration data to the remote client.
+ *
+ * @param  AxesRaw_t* p_axes
+ * @retval None
+ */
+void User_Process(AxesRaw_t* p_axes)
+{
+  /* Check if the user has pushed the button */
+  if(digitalRead(USER_BTN) == RESET)
+  {
+    while (digitalRead(USER_BTN) == RESET);
+
+    /* Update acceleration data */
+    p_axes->AXIS_X += 100;
+    p_axes->AXIS_Y += 100;
+    p_axes->AXIS_Z += 100;
```

Comment:
Same status as demo on L4 IOT Discovery board, the Z axis is not updated on the app side.